test-fixtures/transform/02/outputSucces.json

```diff
@@ -0,0 +1,678 @@
+            {
+              "code": "a",
+              "value": "Minun tarinani /"
+            },
+            {
+              "code": "c",
+              "value": "Michelle Obama ; suomentanut Ilkka Rekiaro."
+            }
+          ]
+        },
+        {
+          "tag": "264",
+          "ind1": " ",
+          "ind2": "1",
+          "subfields": [
+            {
+              "code": "a",
+              "value": "[Helsinki] :"
+            },
+            {
+              "code": "b",
+              "value": "Kustannusosakeyhtiö Otava,"
+            },
+            {
+              "code": "c",
+              "value": "2018."
+            }
+          ]
+        },
+        {
+          "tag": "300",
+          "ind1": " ",
+          "ind2": " ",
+          "subfields": [
+            {
+              "code": "a",
+              "value": "1 verkkoaineisto (EPUB  kb; vastaa painettuna 526 sivua)"
+            }
+          ]
+        },
+        {
+          "tag": "336",
+          "ind1": " ",
+          "ind2": " ",
+          "subfields": [
+            {
+              "code": "a",
+              "value": "teksti"
+            },
+            {
+              "code": "b",
+              "value": "txt"
+            },
+            {
+              "code": "2",
+              "value": "rdacontent"
+            }
+          ]
+        },
+        {
+          "tag": "337",
+          "ind1": " ",
+          "ind2": " ",
+          "subfields": [
+            {
+              "code": "a",
+              "value": "tietokonekäyttöinen"
+            },
+            {
+              "code": "b",
+              "value": "c"
+            },
+            {
+              "code": "2",
+              "value": "rdamedia"
+            }
+          ]
+        },
+        {
+          "tag": "338",
+          "ind1": " ",
+          "ind2": " ",
+          "subfields": [
+            {
+              "code": "a",
+              "value": "verkkoaineisto"
+            },
+            {
+              "code": "b",
+              "value": "cr"
+            },
+            {
+              "code": "2",
+              "value": "rdacarrier"
+            }
+          ]
+        },
+        {
+          "tag": "500",
+          "ind1": " ",
+          "ind2": " ",
+          "subfields": [
+            {
+              "code": "a",
+              "value": "EPUB-julkaisu."
+            }
+          ]
+        },
+        {
+          "tag": "530",
+          "ind1": " ",
+          "ind2": " ",
+          "subfields": [
+            {
+              "code": "a",
+              "value": "Julkaistu myös painettuna."
+            }
+          ]
+        },
+        {
+          "tag": "538",
+          "ind1": " ",
+          "ind2": " ",
+          "subfields": [
+            {
+              "code": "a",
+              "value": "Internet-yhteys, www-selain ; lukuohjelma."
+            }
+          ]
+        },
+        {
+          "tag": "540",
```

Comment:
540 with "Käyttöoikeus Helmet-kirjastokortilla" should have subfield '$5 HELME' , or preferably be dropped.